modules/luabackend/lua_functions.cc

```diff
@@ -230,148 +230,148 @@ void register_lua_functions(lua_State *lua) {
     lua_setglobal(lua, "QTypes");
 }
 
-bool LUABackend::getValueFromTable(lua_State *lua, const std::string& key, string& value) {
-  lua_pushstring(lua, key.c_str());
-  lua_gettable(lua, -2);
+bool LUABackend::getValueFromTable(lua_State *lua_state, const std::string& key, string& value) {
```

Comment:
I'm missing something here, why aren't you also updating the name of the `lua_State` in the functions above?

Reply:
clang did not complain about them.... I guess the difference is function vs method. But it wold be better to do them all indeed.

Reply:
To make it more clear: `lua` is a field in `LUABackend` so for global functions there is no clash.